kore/app/exec/Main.hs

```diff
@@ -430,7 +430,4 @@ mainParseSearchPattern indexedModule patternFileName = do
 makePurePattern
     :: VerifiedKorePattern
     -> CommonStepPattern Object
-makePurePattern pat =
-    case fromKorePattern Object pat of
-        Left err -> error (printError err)
-        Right objPat -> objPat
+makePurePattern = id
```

Comment:
Will the spring cleaning include deleting such now-useless definitions? Like this one and....